crates/humanode-peer/src/chain_spec.rs

```diff
@@ -342,17 +384,15 @@ fn testnet_genesis(
         },
         evm: EVMConfig {
             accounts: {
-                let evm_pot_accounts = vec![(
-                    humanode_runtime::EvmToNativeSwapBridgePot::account_id(),
-                    evm_genesis_account(INITIAL_POT_ACCOUNT_BALANCE),
-                )];
-
-                evm_pot_accounts
+                basic_evm_accounts
                     .into_iter()
+                    .map(|(account_id, balance)| (account_id, evm_genesis_account(balance)))
                     .chain(
```

Comment:
If you swap .chain and .map order you should be able to get rid of one `evm_genesis_account` call literal

Reply:
My bad, thanks - https://github.com/humanode-network/humanode/pull/700/commits/14e6cd096a332224ee79fbe21365b8da22b57154.